srv/components/controller/files/scripts/xml.sh

```diff
@@ -73,10 +73,11 @@ parse_xml()
 ctrl_activity_get()
 {
     _xml_doc=$1
-    ctrl_activity_a="none"
-    ctrl_activity_b="none"
+    ctrl_activity_a=
+    ctrl_activity_b=
     if [[ ! -f $_xml_doc ]]; then
-        echo "ERROR: could not find the file $_xml_doc." | tee -a $logfile 
+        echo "WARNING: could not find the file $_xml_doc, not parsing the progress file" | tee -a $logfile
```

Comment:
Should this be an error?
If file is not found, it could mean some issue with either connectivity or controller cli.

Reply:
No, this is expected during firmware update, there's a window in few update where the 'get progress' command doesn't work and doesn't return the xml output, this means update is still in progress.. so this is not an issue.